src/plugins/UserDataPlugin.js

```diff
@@ -49,33 +49,39 @@ export class UserDataPlugin extends BasePlugin {
     this.client.on('IDBUpdate', this.onIDBUpdate);
     this.client.on('IDBClose', this.onIDBClose);
     this.client.on('IDBGetVersion', this.onIDBGetVersion);
-
   }
 
   /**
    * Handler for the userDataRemove event 
    * @method onUserDataRemove
    * @private
+   * @param {string} name The name of the record to be removed
+   * @param {string} type The type of listener for bellhop to send to
    */
-  onUserDataRemove({ data, type }) { 
```

Comment:
I would leave these all as `data`. When Bellhop packages the "message" for sending it names them specifically `data` and `type`. So this changes might break things.

Reply:
For reference: https://github.com/SpringRoll/Bellhop/blob/main/src/Bellhop.js#L228